doc/03_research/04_requirements/02_informations_from_leaderboard.md

```diff
@@ -0,0 +1,189 @@
+
+<!-- TOC -->
+* [Requirements of Carla Leaderboard](#requirements-of-carla-leaderboard)
+  * [Author](#author)
+  * [Date](#date)
+  * [Prerequisite](#prerequisite)
+  * [Task](#task)
+  * [Participation modalities](#participation-modalities)
+    * [Route format](#route-format)
+    * [Sensors](#sensors)
+  * [Evaluation](#evaluation)
+    * [Main score](#main-score)
+    * [Driving Score for route i](#driving-score-for-route-i)
+    * [Infraction penalty](#infraction-penalty)
+    * [Shutdown criteria](#shutdown-criteria)
+  * [Submission](#submission)
+    * [Sources](#sources)
+<!-- TOC -->
+
+---
+
+## Task
+
+* an autonomous agent should drive through a set of predefined routes
+* for each route:
+  * initialization at a starting point
+  * directed to drive to a destination point
+  * route described by GPS coordinates **or** map coordinates **or** route instructions
+* route situations:
+  * freeways
+  * urban areas
+  * residential districts
+  * rural settings
+* weather conditions:
+  * daylight
+  * sunset
+  * rain
+  * fog
+  * night
+  * more ...
+
+Possible traffic signs (not complete):
+
+* Stop sign
+* Speed limitation
+* Traffic lights
+* Arrows on street
+* Stop sign on street
+
+## Participation modalities
+
+The two modes are **SENSORS** and **MAP**. They differ in the type of input data the agent can request from the platform.
+
+### Route format
+
+Routes consist of tuples of a position and a high level command which should be taken at that point.
+
+There are two different ways to express a route:
+
+First, GPS coordinates, a z component and a route option
+
+```yaml
+[({'z': 0.0, 'lat': 48.99822669411668, 'lon': 8.002271601998707}, RoadOption.LEFT),
+ ({'z': 0.0, 'lat': 48.99822669411668, 'lon': 8.002709765148996}, RoadOption.RIGHT),
+ ...
+ ({'z': 0.0, 'lat': 48.99822679980298, 'lon': 8.002735250105061}, RoadOption.STRAIGHT)]
+
+```
+
+Second, world coordinates and a route option
+
+```yaml
+[({'x': 153.7, 'y': 15.6, 'z': 0.0}, RoadOption.LEFT),
+ ({'x': 148.9, 'y': 67.8, 'z': 0.0}, RoadOption.RIGHT),
+ ...
+ ({'x': 180.7, 'y': 45.1, 'z': 1.2}, RoadOption.STRAIGHT)]
+
+```
+
+**Important:** Distance between route points can be up to hundreds of meters.
+
+High-level commands (rood options) are:
+
+* RoadOption.**CHANGELANELEFT**: Move one lane to the left.
+* RoadOption.**CHANGELANERIGHT**: Move one lane to the right.
+* RoadOption.**LANEFOLLOW**: Continue in the current lane.
+* RoadOption.**LEFT**: Turn left at the intersection.
+* RoadOption.**RIGHT**: Turn right at the intersection.
+* RoadOption.**STRAIGHT**: Keep straight at the intersection.
+
+**Important:** If the semantics of left and right are ambiguous, the next position should be used to clarify the path.
+
+**Important**: You are not allowed to make use of any privilege information offered by the CARLA simulator, including planners or any type of ground truth. Submissions using these features will be rejected and teams will be banned from the platform.
```

Comment:
Does this also count for training data? Or is it possible to train our nets e.g. with the output of the semantic camera?

Reply:
Good question. I'm not quite sure, but certainly it should not be included in the submission as only RGB cameras are allowed. It might falsify our model if we train on data later unavailable, what do you think?

Reply:
Yes that’s right… So more work for us to create the labels ourselves, but at least we are then on the safe side 😄

Reply:
Possibly there are training sets from the previous years

Reply:
@MarcoRiedenauer to follow up on the issue. In my opinion, for training data generation/annotated images, it should be possible to work with everything we have, for the submission we are limited to RGB cameras